go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -172,6 +172,27 @@ func tstWorkflowComplete(t *testing.T) error {
 	return tstWorkflowAction(t, workflowActionComplete, "", "")
 }
 
+// testWorkflowUpdate is a very simple test of the workflow update
+// vtctlclient/vtctldclient command.
+// It performs a non-behavior impacting update, setting tablet-types
+// to primary,replica,rdonly (the only applicable types in these tests).
+func testWorkflowUpdate(t *testing.T) {
+	tabletTypes := "primary,replica,rdonly"
+	// Test vtctlclient first
+	_, err := vc.VtctlClient.ExecuteCommandWithOutput("workflow", "--", "--tablet-types", tabletTypes, "no.exist", "update")
+	require.Error(t, err, err)
+	resp, err := vc.VtctlClient.ExecuteCommandWithOutput("workflow", "--", "--tablet-types", tabletTypes, ksWorkflow, "update")
+	require.NoError(t, err)
+	require.NotEqual(t, "", resp)
+
+	// Test vtctldclient last
+	_, err = vc.VtctldClient.ExecuteCommandWithOutput("workflow", "--keyspace", "noexist", "update", "--workflow", "noexist", "--tablet-types", tabletTypes)
```

Comment:
did you mean to use a different keyspace here? (`noexist` vs `no.exist` above)

Reply:
It's `keyspace.workflow` in vtctlclient.

Reply:
ahh i see, but that would mean that `vtctlclient` is updating `no.exist` and `vtctldclient` would be updating (in the old parlance) `noexist.noexist` right?

Reply:
Yeah. I can use the same for both. Clearly it's causing some questions :-) Will do that now...